src/pages/Portfolio/MyKiva/MyKivaPage.vue

```diff
@@ -198,21 +201,39 @@ const handleSelectedLoan = loan => {
 	fetchLoanUpdates(activeLoan.value.id);
 };
 
-apollo.query({ query: myKivaQuery })
-	.then(result => {
-		userInfo.value = result.data?.my ?? {};
-		lender.value = result.data?.my?.lender ?? null;
-		loans.value = result.data?.my?.loans?.values ?? [];
-		if (loans.value.length > 0) {
+const fetchMyKivaData = () => {
+	return apollo.query({ query: myKivaQuery })
+		.then(result => {
+			userInfo.value = result.data?.my ?? {};
+			lender.value = result.data?.my?.lender ?? null;
+			loans.value = result.data?.my?.loans?.values ?? [];
+			if (loans.value.length > 0) {
 			// eslint-disable-next-line prefer-destructuring
-			activeLoan.value = loans.value[0];
-			fetchLoanUpdates(activeLoan.value.id);
-		}
-	}).catch(e => {
-		logReadQueryError(e, 'MyKivaPage myKivaQuery');
-	});
-
-onMounted(() => {
+				activeLoan.value = loans.value[0];
+				fetchLoanUpdates(activeLoan.value.id);
+			}
+		}).catch(e => {
+			logReadQueryError(e, 'MyKivaPage myKivaQuery');
+		});
+};
+
+const fetchUserPreferences = async () => {
```

Comment:
This method would make more sense named something like `saveMyKivaToUserPreferences`